### PR TITLE
DP-914 - localization import / export

### PR DESCRIPTION
**Pull request by andymantell**

### CODE REVIEW

File: Services/CO.CDP.Localization/StaticTextResource.resx [hunk 1]
```diff
@@ -643,7 +643,7 @@
     <value>Has other significant influence or control</value>
   </data>
   <data name="Supplier_ConnectedEntity_ConnectedEntityControlCondition_HasOtherSignificantInfluenceOrControlHint" xml:space="preserve">
-    <value>Has the right to exercise, or exercises, significant influence or control over the company. As defined in the &lt;a href="https://www.gov.uk/government/publications/guidance-to-the-people-with-significant-control-requirements-for-companies-and-limited-liability-partnerships" target="_blank"&gt;person of significant control (PSC) guidance&lt;/a&gt;.</value>
```
Comment (andymantell) [inline Services/CO.CDP.Localization/StaticTextResource.resx:646]: This space before "guidance" was a `U+202F : NARROW NO-BREAK SPACE [NNBSP]` - I've made it a normal space

File: Services/CO.CDP.Localization/StaticTextResource.resx [hunk 2]
```diff
@@ -1712,7 +1712,7 @@
     <value>For example, ‘You need to change the email address. The domain name in the organisation email address you gave us does not match the organisation name you want to register’.</value>
   </data>
   <data name="Supplementary_CookieDetails_TableRow_Preferences" xml:space="preserve">
-    <value />
+    <value>Stores the cookies accept / reject preference.</value>
```
Comment (andymantell) [inline Services/CO.CDP.Localization/StaticTextResource.resx:1715]: This key was missing a value. It had been duplicated initially, and then has been through many many merge conflicts in this file and it had got lost as a result

File: Services/CO.CDP.Localization/StaticTextResource.resx [hunk 3]
```diff
@@ -277,7 +277,7 @@
     <value>Email address:</value>
   </data>
   <data name="Organisation_ChangeSignInDetails" xml:space="preserve">
-    <value>You can <a class="govuk-link govuk-link--no-visited-state" href="{0}" target="_blank">change your sign in details in your GOV.UK One Login</a>.</value>
+    <value>You can &lt;a class="govuk-link govuk-link--no-visited-state" href="{0}" target="_blank"&gt;change your sign in details in your GOV.UK One Login&lt;/a&gt;.</value>
```
Comment (andymantell) [inline Services/CO.CDP.Localization/StaticTextResource.resx:280]: Fixed a bug here - the link was not showing up in the rendered html at all

**Review by jakzal — APPROVED** [review]
Includes the docs. Perfect 👌

**Review by dharmverma — APPROVED** [review]

**Pull request merged**